.builds/ci.yml

```diff
@@ -31,7 +31,8 @@ tasks:
       for py in "${PYTHON_VERSIONS[@]}"; do
         source ".venv-$py/bin/activate"
         pushd portage
-          time ./setup.py clean install
```

Comment:
Could you bang in a single muon job (look at pax-utils)?

Reply:
Unfortunately, muon does not (yet? patches welcome, usually) support the `import('python')` module for non-trivial uses -- that is, anything other than using find_installation() as an inefficient replacement for `find_program('python3')`.

Reply:
I guess not then. :cry:

Reply:
To be fair it's not that bad... You need python in this case anyway so meson can be bootstrapped with just that python (as a pyz "zipapp" for example). The value of muon is a bit less since you can't knock python out of the dependency loop.

Still better than setuptools. :D